app/app.py

```diff
@@ -30,6 +30,8 @@ async def on_startup(app):
     # by limiting keep-alive, we help prevent errors during RHSvc scale-back.
     conn = TCPConnector(keepalive_timeout=5)
     app.http_session_pool = ClientSession(connector=conn, timeout=ClientTimeout(total=30))
+    # Use below line instead if using a proxy (for AD API etc)
+    # app.http_session_pool = ClientSession(connector=conn, timeout=ClientTimeout(total=30), trust_env=True)
```

Comment:
The second version has the extra param, which I needed to get the GTF api work locally, but it's not needed when deployed - I could remove the comment add add it to the readme instead?
I didn't want to leave in the extra param as it wasn't needed, in case it caused any unexpected side effects

Reply:
It worked locally for me without the extra trust_env parameter, strange.

Reply:
Might well be an issue with my set-up then - I'll remove it